### PR TITLE
migrate majority of routes to use wiki.Handler

### PR DESCRIPTION
**Pull request by joshuabenuck**
This PR migrates all but the ProcessStep routes to use wiki.Handler.

### CODE REVIEW

File: dig.ts [hunk 1]
```diff
@@ -15,96 +16,61 @@ import {
...
-      wiki.serveFile(req,'image/png',path)
-      return true
-    }
-  }
-  if(req.url.startsWith('/assets/')) {
-    let path = join('../seran-dig',req.url)
-    if (existsSync(path)) {
-      wiki.serveFile(req,'text/html',path)
-      return true
-    }
```
Comment (WardCunningham) [inline dig.ts:31]: Need to handle assets too

**Review by WardCunningham — COMMENTED** [review]
I can't figure out how to merge and correct this wilthout triggering a build and deploy with this code missing.

**Comment by WardCunningham** [comment]
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally

Isn't helping. Suggestions?

File: dig.ts (hunk 1 above)
Comment (joshuabenuck) [reply dig.ts:31]: Assets is handled down below.

File: dig.ts [hunk 2]
```diff
@@ -15,96 +16,61 @@ import {
...
-    .toLowerCase()
-}
+handler.route("^/assets/", (req) => {
+  let path = join('../seran-dig',req.url)
+  if (existsSync(path)) {
+    wiki.serveFile(req,'text/html',path)
+    return true
+  }
+  return false
+})
```
Comment (joshuabenuck) [inline dig.ts:37]: Here is the route for assets.

**Comment by joshuabenuck** [comment]
When I need to make changes to a PR, I use the drop down to the right of the merge button. There's an option about checking out the changes locally. It gives a series of git cli commands you can run to test the changes locally. They are a bit different than the ones in the article you linked to earlier.

**Comment by WardCunningham** [comment]
Thank you. That "merge" link is super handy.

**Pull request merged**